connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -389,14 +390,20 @@ async function syncOneFile(
   });
 
   if (documentContent.length <= MAX_DOCUMENT_TXT_LEN) {
+    const parents = await getFileParentsMemoized(
+      connectorId,
+      oauth2client,
+      file,
+      startSyncTs
+    );
     await upsertToDatasource({
       dataSourceConfig,
       documentId,
       documentText: documentContent,
       documentUrl: file.webViewLink,
       timestampMs: file.updatedAtMs,
       tags,
-      parents: [],
+      parents: parents.map((p) => p.id),
```

Comment:
You want to add the file itself as well (I thiiiiiink it's not in the return of that function, is it?)